stories/resizable-card-layout/resizable-card-layout.stories.js

```diff
@@ -0,0 +1,505 @@
+
+Cards can be resized vertically in rows of 1 rem and horizontally in steps of 20 rem (minimum width).
+        `,
+        tags: ['f3', 'theme', 'development'],
+        components: [
+            'resizable-card-layout',
+            'button',
+            'avatar',
+            'badge',
+            'card',
+            'object-status',
+            'numeric-content',
+            'table',
+            'checkbox',
+            'list',
+            'link',
+            'icon'
+        ]
+    }
+};
+
+export const resizeCardLayout = () =>
+    `<div id="grid1" class="fd-resizable-card-layout fd-resizable-card-layout--md fd-custom-resizable-card-layout-flex">
+        <div id="one" class="fd-resizable-card-layout__wrapper card-one-dimensions">
+            <div class="fd-resizable-card-layout__wrapper__resize__handle--vertical"></div>
+            <div class="fd-resizable-card-layout__wrapper__card">
+                <div class="fd-card" role="region" aria-label="List Card Example 1">
+                    <a class="fd-card__header" tabindex="0">
+                        <div class="fd-card__header-text">
+                            <div class="fd-card__title-area">
+                                <div class="fd-card__title">Quick Links</div>
+                                <span class="fd-object-status fd-card__counter">4 of 20</span>
+                            </div>
+                        </div>
+                    </a>
+                    <div class="fd-card__content" role="group" aria-label="Card Content">
+                        <ul class="fd-list fd-list--no-border" role="list">
+                            <li role="listitem" tabindex="0" class="fd-list__item">
+                                <i role="presentation" class="fd-list__icon sap-icon--cart"></i>
+                                <span class="fd-list__title">List item 1</span>
+                            </li>
+                            <li role="listitem" tabindex="0" class="fd-list__item">
+                                <i role="presentation" class="fd-list__icon sap-icon--wrench"></i>
+                                <span class="fd-list__title">List item 2</span>
+                            </li>
+                            <li role="listitem" tabindex="0" class="fd-list__item">
+                                <i role="presentation" class="fd-list__icon sap-icon--leads"></i>
+                                <span class="fd-list__title">List item 3</span>
+                            </li>
+                            <li role="listitem" tabindex="0" class="fd-list__item">
+                                <i role="presentation" class="fd-list__icon sap-icon--batch-payments"></i>
+                                <span class="fd-list__title">List item 4</span>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="fd-resizable-card-layout__wrapper__card__icon-wrapper">
```

Comment:
This can be changed to `fd-resizable-card-layout__icon-wrapper`

Reply:
fixed.